lib/auth/routes_authtype.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function (pluginRoot, server, kbnServer, APP_ROOT, API_ROOT) {
+  /**
+   * Auth type API that returns current auth type configured on Kibana Server.
+   *
+   * GET /api/v1/auth/authtype
+   * Response:
+   *  200 OK
+   *  {
+   *    authtype: saml
+   *  }
+   */
+   server.route({
+       method: 'GET',
+       path: `${API_ROOT}/auth/authtype`,
+       handler: (request, h) => {
+           let authtype = server.config().get('opendistro_security.auth.type');
+           if (authtype === "") {
+             authtype = "basicauth";
+           }
+
+           return { authtype };
```

Comment:
Description says that output should look like:
```
{
  authtype: saml
}
```

Can you confirm that this is how the output looks like ?  
I don't see any key-value in the return statement. 
Also, I expect that value is a string "saml" (from comment it does not look like a string).

Should the return statement be the following ?
```
return {authtype: authtype};
```

Reply:
Hi @dinusX , 

This is a shorthand in ES6. If key and value are the same you can replace `{authtype: authtype}` with `{authtype}`. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New%20notations%20in%20ECMAScript%202015)


This is the full request - 

```
curl localhost:5601/api/v1/auth/authtype 
{"authtype":"basicauth"}
```